src/base/http/server.cpp

```diff
@@ -62,7 +62,8 @@ void Server::enableHttps(const QSslCertificate &certificate, const QSslKey &key)
 void Server::disableHttps()
 {
     m_https = false;
-    m_certificate.clear();
+    foreach (QSslCertificate c, m_certificates)
```

Comment:
@evsh is right.  m_certificates.clear() is enough here. But then it needs to check m_certificates.size() before call m_certificates.first().